js/stdlib/ts/ETH_like_compiled.ts

```diff
@@ -140,11 +140,17 @@ const T_Bytes = (len:number): ETH_Ty<CBR_Bytes, ETH_Bytes> => {
   const me = {
     ...CBR.BT_Bytes(len),
     munge: ((bv: CBR_Bytes): ETH_Bytes => {
-      const bs = Array.from(ethers.utils.toUtf8Bytes(bv));
+      const ubs = bv.startsWith('0x')
```

Comment:
If you do this, you have to update the docs to say the value will be 0x'd if it isn't utf8, but that is backwards incompatible